workflows/gen_crates.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import pathlib
+
+# pip install 'rocrate==0.4.0'
+from rocrate.rocrate import ROCrate
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+
+ROOT = THIS_DIR / "sars-cov-2-variant-calling"
+
+GH_URL = "https://api.github.com"
+GH_RESOURCE = "repos/galaxyproject/iwc/actions/workflows/workflow_test.yml"
```

Comment:
What's the purpose of this ?

Reply:
These end up in the `TestInstance` entity as the `"url"` and `"resource"` properties:

```json
{
    "@id": "#test1_1",
    "@type": "TestInstance",
    "name": "test1_1",
    "runsOn": {
        "@id": "https://w3id.org/ro/terms/test#GithubService"
    },
    "url": "https://api.github.com",
    "resource": "repos/galaxyproject/iwc/actions/workflows/workflow_test.yml"
}
```

LifeMonitor [reads them](https://crs4.github.io/life_monitor/workflow_testing_ro_crate) to find out where to look for test build information.

Reply:
Ah, excellent, that is very nice!